atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -344,6 +344,34 @@ void WebFrame::SetIsolatedWorldHumanReadableName(int world_id,
       world_id, blink::WebString::FromUTF8(name));
 }
 
+void WebFrame::SetIsolatedWorldInfo(int world_id, mate::Arguments* args) {
+  if (args->Length() < 1) {
```

Comment:
I'm pretty sure calling `GetNext` will automatically throw an error if their is no "next" arg.  Can we remove this check so that insufficient arg error messages are consistent

Reply:
We can definitely remove this if you do the suggested change above

Reply:
Done! Thanks! 🙇